firmware/quadruna/VC/src/app/states/app_driveState.c

```diff
@@ -85,7 +86,9 @@ static void driveStateRunOnTick1Hz(void)
 static void driveStateRunOnTick100Hz(void)
 {
     // All states module checks for faults, and returns whether or not a fault was detected.
-    const bool all_states_ok = app_allStates_runOnTick100Hz();
+    const bool any_board_has_fault = app_boardFaultCheck();
+    const bool inverter_has_fault  = app_inverterFaultCheck();
+    const bool all_states_ok       = !(any_board_has_fault || inverter_has_fault);
```

Comment:
this sort of pattern is used a lot, perhaps think about making it also a function? not entirely mandatory.